README.md

```diff
@@ -1,7 +1,7 @@
 # Visual Studio Code Smart Drupal Snippets
 
 This extension adds rich language support for the
-[Drupal 9.3.x Hooks API](https://api.drupal.org/api/drupal/core%21core.api.php/group/hooks/9.3.x)
+[Drupal 9.4.x Hooks API](https://api.drupal.org/api/drupal/core%21core.api.php/group/hooks/9.4.x)
```

Comment:
Might be a good idea just to use the version-less url

https://api.drupal.org/api/drupal/core%21core.api.php/group/hooks

Reply:
Sure, I'll make the changes.

Thanks for being so efficient in reviewing the PR!